pkg/tfgen/test_data/parse-imports/docker_container-expected.md

```diff
@@ -0,0 +1,39 @@
+## Import
+
+```sh
+#!/bin/bash
+$ pulumi import docker:index/container:Container foo id
+```
+
+### Example
+
+Assuming you created a `container` as follows
+
+```sh
+#!/bin/bash
+docker run --name foo -p8080:80 -d nginx 
+# prints the container ID 
+9a550c0f0163d39d77222d3efd58701b625d47676c25c686c95b5b92d1cba6fd
+```
+
+you provide the definition for the resource as follows
+
+```terraform
```

Comment:
We do not, as a rule, display Terraform code on our documentation. This code block would have to undergo example translation using `pulumi convert` as provided by this file's functions: https://github.com/pulumi/pulumi-terraform-bridge/blob/master/pkg/tfgen/convert_cli.go

Reply:
This is not displayed - it is converted into provider code in next step somewhere - in any case docker doc ended being correct - this terraform block was converted into multiple code blocks.